lib/versions.js

```diff
@@ -6,6 +6,21 @@ import { HttpClient } from "@actions/http-client";
 export async function determineVersion(version, { repo, nightliesRepo }) {
   version = version?.trim();
 
+  const toolVersionsPathInput = core.getInput("tool-versions");
```

Comment:
Can we move this call to `main` function? So that all `getInput`s will be colocated